.eslintrc

```diff
@@ -70,6 +70,36 @@
         "import/first": 0,
       },
     },
+    {
+      "files": [
+        "src/Components/Intercom.js",
+      ],
+      "rules": {
+        "class-methods-use-this": 0,
+      },
+    },
+    {
+      "files": [
+        "src/intercom.js",
```

Comment:
I suggest to use .eslintignore for this.

Reply:
Yeah, nice idea!